### ISSUE TITLE
MenuController UX Consistency

### ISSUE THREAD

**Issue by Huddie** [GitHawkApp/GitHawk#2270]
So I recently discovered that long pressing on the time since (now, 2h, 1yr) the post opens a UIMenuController with the full date which that post when sent (Really cool!)

Just tapping on the sent with githawk icon opens a UIMenuController saying “Sent with GitHawk”

I think we should choose either long press or tap to active these “secret” pop ups.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Good feedback. Let’s go with tap

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Huddie** [comment]
@rnystrom working on #2271 I realized that a few things. 
1. Start typing out a comment and while the keyboard is active, tap the timestamp above (rnystrom added design xx minutes ago). I’m getting a weird situation where the menu controller has [“select”, “select all”, “paste”, timestamp]
2. With the keyboard resigned it’s sometimes difficult to get the timestamp mentioned above to show the menu when tapped. In order to get it working first tap the timestamp near the githawk badge and then tap it. 
3. With keyboard active press and hold the comment textfield. Again you should see [“select”, “select all”..., timestamp] which doesn’t make sense 

Lmk if that’s a bit hard to follow. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Huddie** [comment]
I think I could add becomeFirstResponder to resign keyboard to fix the paste/select/select all issue.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Check out how we resign active in other places by calling `view.endEditing(true)` in a lot of places
